libraries/AP_Scripting/examples/analog_input_and_GPIO.lua

```diff
@@ -0,0 +1,32 @@
+-- This script is an example of reading a analog pin
+
+-- load the analog pin
+local analog = hal.analog_pin()
+
+-- load a pwm pin
+local pwm_in = PWMSource()
+
+analog:set_pin(13) -- typically 13 is the battery input
+pwm_in:set_pin(50) -- AUX 1
```

Comment:
set_pin can fail if ISR not available on this pin, maybe a comment and a print if failed?

Reply:
also will fail if BRD_PWM_COUNT is not correct

Reply:
Added a print if it fails, and another pin that trys aux 6 and always fails (on a Cube), also added a comment about `BRD_PWM_COUNT`